src/components/challenges/active.js

```diff
@@ -0,0 +1,32 @@
+import classNames from './styles.module.scss';
+
+const Active = ({ content }) => {
+  return (
+    <div className={classNames.boxConent}>
+      <p className={classNames.heading}>{content.title}</p>
+      <p>
+        <span className={classNames.description}>Level</span>:
+        <span className={classNames.descValue}>{content.level}</span>
+      </p>
+      <p>
+        <span className={classNames.description}>Challenge Started</span>:
+        <span className={classNames.descValue}>{content.startDate}</span>
+      </p>
+      <p>
+        <span className={classNames.description}>Challenge Ends</span>:
+        <span className={classNames.descValue}>{content.endDate}</span>
+      </p>
+      <p>
+        <span className={classNames.description}>Active Participants</span>:
+        <span
+          className={classNames.descValue}
+        >{`${content.participantsCnt} Members`}</span>
+      </p>
```

Comment:
Seeing a lot of repeated code of the format:

```jsx
return (
      <p>
        <span className={classNames.description}>{desc}</span>:
        <span className={classNames.descValue}>{val}</span>
      </p>
)
```

What are your thoughts on making it a reusable function @sumitd94?

Reply:
Done
Ref : https://github.com/Real-Dev-Squad/website-roadmap/pull/8/commits/0e9582b11de84acbb05306c5964d59429c572733